policy_agent/policy_agent.py

```diff
@@ -0,0 +1,643 @@
+#!/usr/bin/python
+import os
+import sys
+import time
+import json
+import Queue
+import logging
+import requests
+import pycalico
+
+from threading import Thread
+from subprocess import check_output
+from contextlib import closing
+from pycalico.datastore_datatypes import Rules, Rule
+from pycalico.datastore_errors import ProfileNotInEndpoint, ProfileAlreadyInEndpoint
+from pycalico.datastore import DatastoreClient, PROFILE_PATH
+
+KUBE_API_ROOT = os.environ.get('KUBE_API_ROOT',
+                               'http://localhost:8080/api/v1/')
+
+ANNOTATION_NAMESPACE = "projectcalico.org/"
+EPID_ANNOTATION_KEY = "%sendpointID" % ANNOTATION_NAMESPACE
+
+POLICY_LOG_DIR = "/var/log/calico/policy"
+POLICY_LOG = "%s/calico.log" % POLICY_LOG_DIR
```

Comment:
I don't think we want to keep calling all of our logs `calico.log` but in a different location.

This should probably be "/var/log/calico/kubernetes/policy-agent.log"